active/radiobuttons/RadioButtons_Spec.md

```diff
@@ -0,0 +1,252 @@
+
+- The up or down arrow keys will move to the "next" or "previous" logical item (defined in markup)
+- The left or right arrow keys will move spatially
+
+![alt text](keyboardnav_singlecol.png)
+
+This means that in purely vertical RadioButton group lists, left/right arrow keys will not do anything, but up/down will still navigate as expected. However, in purely horizontal RadioButton group lists, left and right will navigate the same way as up and down.
+
+>This behavior is defined more below when we discuss multi-column RadioButton group lists.
+
+## Vertical and horizontal RadioButton lists
+In instances where there is more that one column of RadioButton items within a single RadioButtons list control, the list will flow a column-major layout.
+
+|MaximumColumns not specified <br>OR<br> MaximumColumns = 1 |  MaximumColumns = 4 |
+|:--:|:--:|
+| ![alt text](singlecolumn_example.png) | ![alt text](multicolumn_example_1.png) |
+
+```xml
+<RadioButtons Header="Select Number" MaximumColumns="2">
+    <x:String>1</x:String>
+    <x:String>2</x:String>
+    <x:String>3</x:String>
+    <x:String>4</x:String>
+</RadioButtons>
+```
+
+By default, the RadioButtons list will fill itself vertically, meaning if no ``MaximumColumns`` value is set, it will be assumed to be 1 and orient itself vertically.
+
+### Multiple columns (uncommon case)
+
+If you specify a ``MaxiumumColumns`` value that is not equal to the number of items in the RadioButtons list, the list will arrange  itself in column-major order and put any uneven/remaining items in the first column, in the case of an uneven ``MaximumColumns`` value or list items defined.
+
+|MaximumColumns = 3|
+|:--|
+|![alt text](multicolumn_example_2.png)|
+
+```xml
+<RadioButtons Header="Select Number" MaximumColumns="3">
+    <x:String>1</x:String>
+    <x:String>2</x:String>
+    <x:String>3</x:String>
+    <x:String>4</x:String>
+    <x:String>5</x:String>
+    <x:String>6</x:String>
+    <x:String>7</x:String>
+</RadioButtons>
+```
+
+#### Keyboard navigation with multiple columns
+The keyboarding behavior is the same as the single-column navigation, it just wraps to the next column when there is more than one defined.
+
+![alt text](keyboardnav_multicol.png)
+
+## Selection follows Focus
+When you are navigating a RadioButtons list via the keyboard, as focus is placed on a previous or next RadioButton item, that item will also get selected/checked. This means the previous item that was selected/checked will be unselected, and the currently focused one will be instead.
+
+|Before keyboard navigation | After keyboard navigation|
+|:--|:--|
+| ![alt text](2selected_nonav.png) | ![alt text](3selected_yesnav.png)|
+| Focus is on the "2" RadioButton, and it is shown as selected | The down or right arrow key has been pressed, so focus was moved to the "3" RadioButton, thus selected 3 and unselected 2. |
```

Comment:
>right [](start = 77, length = 5)

In this single column case, pressing right should not change focus, correct?

Reply:
I believe that that is what we settled on in the last meeting @chigy to confirm.

Once I get confirmation, I'll edit this API.

Reply:
> I believe that that is what we settled on in the last meeting @chigy to confirm.
> 
> Once I get confirmation, I'll edit this API.

Based on the old thread, this is what we settled last so sounds right to me.
-	Have up arrow move previous and down arrow move next
-	Left and right arrow keys move spatially